### PR TITLE
replace labels to const to reduce hardcode

### PR DESCRIPTION
**Pull request by yeya24**
In cluster/delegate.go, I change the label name full_state and update to const value.
Signed-off-by: yeya24 <ben.ye@daocloud.io>

### CODE REVIEW

File: cluster/delegate.go [hunk 1]
```diff
@@ -25,8 +25,12 @@ import (
 )
 
 // Maximum number of messages to be held in the queue.
```
Comment (mxinden) [inline cluster/delegate.go]: This comment would need to be moved below `const (`.

Comment (yeya24) [reply cluster/delegate.go]: done

File: cluster/delegate.go [hunk 2]
```diff
@@ -24,9 +24,13 @@ import (
 	"github.com/prometheus/client_golang/prometheus"
 )
 
-// Maximum number of messages to be held in the queue.
-const maxQueueSize = 4096
+const (
+	// Maximum number of messages to be held in the queue.
+	maxQueueSize = 4096
```
Comment (mxinden) [inline cluster/delegate.go]: Can you run gofmt on this?

```diff
diff -u ./cluster/delegate.go.orig ./cluster/delegate.go
--- ./cluster/delegate.go.orig	2019-01-25 18:08:08.853426105 +0000
+++ ./cluster/delegate.go	2019-01-25 18:08:08.853426105 +0000
@@ -27,10 +27,10 @@
 const (
 	// Maximum number of messages to be held in the queue.
 	maxQueueSize = 4096
-	fullState = "full_state"
-	update = "update"
-
+	fullState    = "full_state"
+	update       = "update"
 )
+
 // delegate implements memberlist.Delegate and memberlist.EventDelegate
 // and broadcasts its peer's state in the cluster.
 type delegate struct {
```

That should fix the CI failures.

Comment (yeya24) [reply cluster/delegate.go]: done

**Pull request merged**